.github/workflows/tests.yml

```diff
@@ -16,6 +16,11 @@ jobs:
           POSTGRES_DB: court-share
     steps:
       - uses: actions/checkout@v2
+      - name: Log in to Docker Hub
```

Comment:
I had to add a docker hub account into the workflow due to issues with Anonymous Rate Limits that causes the test to fail.